spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/refresher_spec.rb

```diff
@@ -3,12 +3,28 @@
     expect(described_class.ems_type).to eq(:ibm_cloud_power_virtual_servers)
   end
 
+  specs = YAML.load_file(File.join(__dir__, "state.yml"))
+  specs['resources'].each do |key, _value|
+    case key['type']
+    when 'ibm_pi_instance'
+      let(:instance_data) { key['instances'][0]['attributes'] }
+    when 'ibm_pi_image'
+      let(:instance_image) { key['instances'][0]['attributes'] }
+    when 'ibm_pi_network'
+      let(:public_network) { key['instances'][0]['attributes'] } if key['name'] == 'public_network'
+      let(:power_network) { key['instances'][0]['attributes'] } if key['name'] == 'power_network'
+    when 'ibm_pi_volume'
+      let(:volume_data) { key['instances'][0]['attributes'] }
+    end
+  end
+
   context "#refresh" do
     let(:ems) do
-      uid_ems  = "8fa27c40-827c-4568-8813-79b398e9cd27"
+      uid_ems  = Rails.application.secrets.ibm_cloud_power[:power_instance_id]
```

Comment:
Looks like this is now named `:cloud_instance_id`
```suggestion
      uid_ems  = Rails.application.secrets.ibm_cloud_power[:cloud_instance_id]
```

Reply:
Thanks Adam. I just forgot to make this change.I have corrected it.